src/main/java/seedu/address/logic/commands/ResumePreviewCommand.java

```diff
@@ -79,7 +79,7 @@ private String getDataFromResume(Resume resume, Model model) {
         data.append("\n=========================\n")
                 .append("PERSONAL DETAILS\n")
                 .append("=========================\n\n")
-                .append(model.getUser().toPreview())
+                .append(model.getUser().getInternalPerson().toPreview())
```

Comment:
Can we create a getPerson method and getObservablePerson method separately to avoid try to follow law of Demeter?